editoast/editoast_schemas/src/rolling_stock/rolling_resistance.rs

```diff
@@ -8,38 +9,46 @@ editoast_common::schemas! {
     RollingResistancePerWeight,
 }
 
+#[editoast_derive::annotate_units]
 #[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize, ToSchema, Derivative)]
 #[derivative(Hash)]
 #[serde(deny_unknown_fields)]
 #[allow(non_snake_case)]
 pub struct RollingResistance {
     #[serde(rename = "type")]
     pub rolling_resistance_type: String,
-    /// Solid friction in N
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
-    pub A: f64,
-    /// Viscosity friction in N/(m/s)
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
-    pub B: f64,
-    /// Aerodynamic drag in N/(m/s)²
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
-    pub C: f64,
+    /// Solid friction
+    #[derivative(Hash(hash_with = "newton::hash"))]
+    #[serde(with = "newton")]
+    pub A: Force,
+    /// Viscosity friction in N/(m/s) – N = kg⋅m⋅s⁻²
+    #[derivative(Hash(hash_with = "kilogram_per_second::hash"))]
+    #[serde(with = "kilogram_per_second")]
+    pub B: MassRate,
+    /// Aerodynamic drag in N/(m/s)² – N = kg⋅m⋅s⁻²
+    #[derivative(Hash(hash_with = "kilogram_per_meter::hash"))]
+    #[serde(with = "kilogram_per_meter")]
+    pub C: LinearMassDensity,
```

Comment:
I thought the `-` was a minus and got confused, how about this comment ? Feel free to make the change or not it's just a suggestion.
```suggestion
    /// Solid friction
    #[derivative(Hash(hash_with = "newton::hash"))]
    #[serde(with = "newton")]
    pub A: Force,
    /// Viscosity friction in N/(m/s) = kg/s
    #[derivative(Hash(hash_with = "kilogram_per_second::hash"))]
    #[serde(with = "kilogram_per_second")]
    pub B: MassRate,
    /// Aerodynamic drag in N/(m/s)² = kg/m
    #[derivative(Hash(hash_with = "kilogram_per_meter::hash"))]
    #[serde(with = "kilogram_per_meter")]
    pub C: LinearMassDensity,
```
Though the `MassRate` and `LinearMassDensity` are correct i don't think they help understanding what those parameters are. I don't know the purpose of that part of the code. If it's to summarize the unit under it's simplest form then i agree, if not we might be able to find more understandable units. But if i understand well it's the first situation, right ?

Reply:
The main goal is to guarantee unit homogeneity (it is now impossible to add an acceleration with a speed)

Documenting is a nice bonus. I made aliases, it should be better now